tests/Wrapper/tests.ts

```diff
@@ -0,0 +1,452 @@
+//@ts-nocheck
+
+import { time } from "@openzeppelin/test-helpers";
+
+import { expect } from "chai";
+import { ethers } from "hardhat";
+
+import Wallet from "../helpers/Wallet";
+import { ErrorMessages, TOKENS, Events } from "../helpers/types";
+import {
+  TEST_NAME,
+  TEST_SYMBOL,
+  INVALID_TOKEN_ID,
+  ETH0,
+  ETH1,
+  ETH2,
+  ETH3,
+  ETH4,
+  TAX_DENOMINATOR,
+} from "../helpers/constants";
+import { now } from "../helpers/Time";
+import { taxationPeriodToSeconds } from "../helpers/utils";
+import type { TestConfiguration } from "../helpers/types";
+
+//$ Tests
+
+async function tests(config: TestConfiguration): Promise<void> {
+  //$ Constants
+  const GLOBAL_TRX_CONFIG = {
+    gasLimit: 9500000, // if gas limit is set, estimateGas isn't run superfluously, slowing tests down.
+  };
+  let DEPLOYER = "0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266";
```

Comment:
Assuming this is the contract deployer's address, use `signers[0].address`

Reply:
Fixed